exploring_UV_extinction_curves/Exploring_UV_extinction_curves.ipynb

```diff
@@ -0,0 +1,1662 @@
+{
```

Comment:
<p>This star is known to be reddened by dust. Can you add anything else about the physical situation? Why is this star reddened?</p><p> </p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/ScienceBetter/notebooks-stsci/pull/9/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='11'/>